tests/cypress/specs/scrapers.cy.js

```diff
@@ -0,0 +1,26 @@
+const endpoints = ["auth", "cgi", "eligibility/app", "login", "sample/api"];
+const files = [".env", "wp-admin/login.php", "data.json", "secrets/prod.yaml"];
+const targets = endpoints.concat(files);
+
+const visit = (partial_path) => {
+  return cy.request({
+    method: "GET",
+    url: `/${partial_path}`,
+    // allow cypress to continue on 404
+    failOnStatusCode: false,
+  });
+};
+
+const NOT_FOUND = 404;
+
+describe("Scraper filtering spec", () => {
+  targets.forEach((target) => {
+    it(`nginx returns 404 for known scraper target pattern: /${target}`, () => {
+      visit(target).then((res) => {
+        expect(res.status).to.eq(NOT_FOUND);
+        expect(res.body).to.contain("nginx");
```

Comment:
I'm running Benefits using `docker compose up client`.

This line fails for me, but I don't see how my set up would have a different result from the GitHub Action run which passed: https://github.com/cal-itp/benefits/actions/runs/4704714930/jobs/8344585869#step:4:211

If it passes for you @thekaveman , maybe we don't worry about my failure?

![image](https://user-images.githubusercontent.com/25497886/232884001-243e94d5-62cc-47a2-8ff7-ca337ff74205.png)

Reply:
One thing I notice is you are running on `localhost:11369` -- seems like your `.env` with secrets is still coming into play.

Try this:

```bash
mv .env .env.bak
touch .env
docker compose up -d client
```

Then run tests.

Reply:
I recall that a reCAPTCHA config breaks the Cypress tests.

Reply:
Nice catch! I rebuilt `client`, and now these tests pass ✅ 

![image](https://user-images.githubusercontent.com/25497886/232898390-a7384b1a-7462-43db-9fe3-7c67f265734e.png)

Reply:
Just a note that you don't actually have to rebuild if all you're doing is changing the `.env` -- you can just `docker compose down` and `docker compose up` to recreate the _container_ but using the same _image_.

`.env` only comes into play when _running the container_.

Reply:
Got it, makes sense 👍